pkg/worker/worker.go

```diff
@@ -67,6 +89,19 @@ func (w *Worker) Work() {
 	*err := mgr.RunWithContext(ctx)
 	******************************/
 
+	/**********************
+	* // Future Enhancement
```

Comment:
Mark with // TODO: ... for IDE capture

Reply:
Future Enhancement: can be captured with IDE, (I for example use [TODO-TREE](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree)). There are other instances of this in the codebase. I'd like to leave this as is.